platform/index.md

```diff
@@ -1,18 +1,70 @@
 ---
-title: Platform
-summary: Platform level index for articles, guidance and documentation
+title: Particular Service Platform Overview
+summary: A short overview of Particular Platform
+tags: 
+- Particular Service Platform, NServiceBus, ServiceMatrix, ServiceControl, ServiceInsight, ServicePulse
```

Comment:
yaml lists should either be newlined with a dashed or comma delimited. not both

i prefer the newline syntax